opendatahub-io/data-science-pipelines-operator
# Adding sample iris pipeline and new base image

## Description of your changes:


Adding an example iris training pipeline, that we could point to for performance testing, or any other purposes. Also added a new base image along with a Dockerfile from which it was built.

## Testing instructions

Deploy DSPv2. On the DSP UI, create a Pipeline using this Pipeline Spec and trigger a Pipeline Run. Make sure the run goes through successfully.

## Checklist
